app/views/markets/mkt_home.html.erb

```diff
@@ -0,0 +1,36 @@
+<h1> Welcome, <%= @market.name %> </h1>
+
+<table>
+  <tr>
+    <th>Name</th>
+    <th>Address</th>
+    <th>City</th>
+    <th>County</th>
+    <th>State</th>
+    <th>Zip</th>
+    <th>Vendors</th>
+  </tr>
+
+  <tr>
+    <td><%= @market.name %></td>
+    <td><%= @market.address %></td>
+    <td><%= @market.city %></td>
+    <td><%= @market.country %></td>
+    <td><%= @market.state %></td>
+    <td><%= @market.zip %></td>
+    <td>
+      <ul>
+        <% @market.vendors.each do |vendor| %>
+        <li><%= vendor.name %> <%= link_to "Delete", vendor, method: :delete %></li>
```

Comment:
I love how concise this delete link code is. However, I think we should probably also have the `data: { confirm: "Are you sure you want to delete this vendor?" }` bit from previous situations where we've used a delete link/button.

It's always a good idea to get a confirmation before actually deleting something.